gef-ui/styles/styles.scss

```diff
@@ -101,7 +100,7 @@ $govuk-new-link-styles: true;
         margin-top: 0;
       }
       white-space: pre-line;
-      word-break: break-word;
+      @include govuk-text-break-word();
```

Comment:
does this need an empty line before it? codacy comment

Reply:
I'm not seeing any Codacy comments?

Reply:
weird, there was just one 30mins ago which has disappeared

Reply:
![image](https://github.com/user-attachments/assets/c2a9f18d-be83-494b-a835-7739c298ecd7)

Reply:
Not sure, maybe something to do with previous merge conflicts.